pom.xml

```diff
@@ -1364,7 +1364,7 @@ flexible messaging model and an intuitive client API.</description>
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-surefire-plugin</artifactId>
-          <version>2.21.0</version>
+          <version>3.0.0-M3</version>
```

Comment:
it works with mvn-3.3 right?

Reply:
Yes, it just requires at least Maven 3.0 : https://maven.apache.org/surefire/maven-surefire-plugin/